sandbox.c

```diff
@@ -128,7 +129,17 @@ open_landlock(void)
 					LANDLOCK_ACCESS_FS_MAKE_REG,
 	};
 
-	return landlock_create_ruleset(&attr, sizeof(attr), 0);
+	fd = landlock_create_ruleset(&attr, sizeof(attr), 0);
+	if (fd == -1) {
+		switch (errno) {
+		case ENOSYS:
+		case ENOTSUPP:
```

Comment:
```
telescope> sandbox.c: In function 'open_landlock':
telescope> sandbox.c:136:8: error: 'ENOTSUPP' undeclared (first use in this function); did you mean 'ENOTSUP'?
telescope>   136 |   case ENOTSUPP:
telescope>       |        ^~~~~~~~
telescope>       |        ENOTSUP
```

Reply:
I can't type today.  Sorry for wasting your time, now it should be corrected.  (I didn't had a linux env around and thought it was easy enough to get it right at the first try.)